suite-native/app/ios/Podfile.lock

```diff
@@ -611,7 +612,7 @@ DEPENDENCIES:
   - react-native-mmkv (from `../../../node_modules/react-native-mmkv`)
   - react-native-restart (from `../../../node_modules/react-native-restart`)
   - react-native-safe-area-context (from `../../../node_modules/react-native-safe-area-context`)
-  - "react-native-skia (from `../../../node_modules/@shopify/react-native-skia`)"
+  - "react-native-skia (from `../node_modules/@shopify/react-native-skia`)"
```

Comment:
path doesnt seems right, it should be `../../../` please call `yarn` and then again pod install

Reply:
https://github.com/trezor/trezor-suite/pull/10202/commits/92ead3077d151da55719b7ddbbf314a5c8f67250